app/helpers/basic_auth_helper.rb

```diff
@@ -0,0 +1,13 @@
+module BasicAuthHelper
+  def blanket_basic_auth
+    if FEATURES.dig(:basic_auth, :enabled)
+      authenticate_or_request_with_http_basic do |username, password|
+        basic_auth_check = username == FEATURES.dig(:basic_auth, :username) &&
```

Comment:
All these `=`s are tricky to parse in the absence of parens!

Reply:
I'll add some!